service/src/test/java/bio/terra/workspace/app/configuration/external/controller/WorkspaceApiControllerTest.java

```diff
@@ -77,6 +99,13 @@ public void setup() throws InterruptedException {
             new UserStatusInfo()
                 .userEmail(USER_REQUEST.getEmail())
                 .userSubjectId(USER_REQUEST.getSubjectId()));
+
+    when(mockFeatureConfiguration.isTpsEnabled()).thenReturn(true);
```

Comment:
Should we enable TPS in `application-test.yml`?

Reply:
we probably should for the sake of reminding future test writers this is on, but for these tests I think it's valuable to verify the different behavior with it off/on, so I think the mocks help